docs/explanations/architecture/app-image-export.md

```diff
@@ -0,0 +1,11 @@
+# Application image export
+
+The `epinio app export` command can be used to export all the needed part of an application (the helm chart, its values and the container image).
+
+The [`skopeo`](https://github.com/containers/skopeo) cli is used to download the image. This was choose over the standard `docker` cli because it doesn't need a deamon to run, it doesn't require root for most of its operations and it's OCI compliants.
```

Comment:
```suggestion
The [`skopeo`](https://github.com/containers/skopeo) cli is used to download the image from the registry. This was chosen over the standard `docker` cli because it doesn't need a demon to run, nor does it require root for most of its operations, and it is OCI compliant.
```